components/tools/OmeroJava/test/integration/chown/PermissionsTest.java

```diff
@@ -19,11 +19,14 @@
 
 package integration.chown;
 
+import static omero.rtypes.rstring;
```

Comment:
@jburel @joshmoore @mtbc : I removed the static import in this commit https://github.com/openmicroscopy/openmicroscopy/pull/4711/commits/05a01c617649eb27727c24b7ef1ac025fa55ebe9